index.html

```diff
@@ -182,6 +182,20 @@ <h1 id="skill-text">Skills</h1>
             </div>
         </div>
     </section>
+<section>
+    <div class="contact-me">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <input type="text" name="name" class="text" placeholder="Full name" max="30" required>
+            <img src="./assets/images/Container.svg"  class="image-line" alt="image-line">   
+            <input type="email" class="text" placeholder="Email address" required>
+            <img src="./assets/images/Container.svg"  class="image-line2" alt="image-line">   
+            <textarea name="comment" class="input-text" placeholder="Enter text here" max="500"></textarea>
+            <button type="button">Get in touch</button>
+            <img src="./assets/images/Container.svg"  class="hl" alt="image-line">   
+            <img src="./assets/images/Social Media.svg" class="media" alt="media">  
+        </div>
+    </section>
```

Comment:
- [ ] Hello and nice job, however, the contact form you have implemented is not the one suggested by the  curriculum so please use [ this reference](https://formspree.io/html/) to have the correct form in your portfolio